packages/rollup-config-nlx/index.ts

```diff
@@ -65,9 +65,7 @@ export default function buildconfig({
         commonjs(), // needed to bundle node modules that use cjs
         json(), // needed to fetch package version from package.json
         nodeResolve({ browser: true }), // bundles packages from node_modules
-        nodePolyfills({
-          include: "../../node_modules/**/*.js",
```

Comment:
before #119 this value was "../**/node_modules/**/*.js" (presumably taken from https://github.com/ionic-team/rollup-plugin-node-polyfills/issues/17#issuecomment-771803747).

I believe in 119 I changed it to `../..` to point to the proper `node_modules` path which may have caused this regression.

I still don't understand why `../**/node_modules` is relevant. I believe in the comment linked above, it's for a different monorepo setup where `node_modules` are symlinked into package directories.

In any case, the change to `../../` broke things. I thought I had confirmed that it worked by diffing umd output files but I certainly didn't test every package.

Do you understand what the previous `include` param was doing? If not I'd rather drop it altogether.